NotesApplication/NotesApplication/Migrations/20221014095945_second.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace NotesApplication.Migrations
+{
+    public partial class second : Migration
```

Comment:
Ime migracije bi uvek trebalo da opisuje ono sto se u njoj tracno radi, second nam ne govori nista. Init moze da bude izuzetak. Pored toga, imena migracija treba da pocinju velikim slovom, kao sto i prvo slovo u svakoj reci treba da bude veliko. Ako se koriste skracenice, opet je samo prvo veliko, ostala mala. Izuzetak moze da bude samo kada je skracenica od 2 slova, al i to je retko slucaj. Procitaj malo o naming convention-ima vezano i za .net i za ef core.

(Ovo sada ne moras da ispravljas, morala bi da brises migraciju pa ponovo da je pravis, al za ubuduce da znas da je imenovanje jako bitno)